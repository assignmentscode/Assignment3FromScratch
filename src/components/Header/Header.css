html, body {
    height: 100%;
    width: 100%;
}
.header {
    width : 100%;
    height: 100px;
}
.first-row {
    display: flex;
    justify-content: space-between;
    padding-top: 1%;
}
h1 {
    color : white;
    margin: 0;
    border: 2px solid white;
    align-self: center;
    border-radius: 80%;
}
.header-line {
    background-color: white;
    width:45%;
    height:4px;
    align-self: center;
}
h2 {
    color: white;
    text-align: center;
}