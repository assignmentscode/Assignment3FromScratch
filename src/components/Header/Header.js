import React from 'react';
import './Header.css';

const Header = () => (
  <div className="header">
    <div className="first-row">
      <div className="header-line" />
      <h1>BS</h1>
      <div className="header-line" />
    </div>
    <h2>The Book Shelf</h2>
  </div>
);
export default Header;
