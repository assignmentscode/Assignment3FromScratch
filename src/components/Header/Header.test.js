import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  it('should render a header', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  const wrapper = shallow(<Header />);
  it('should have 3 listed items in nav bar', () => {
    expect(wrapper.find('div').getElements().length).toEqual(4);
  });
});
