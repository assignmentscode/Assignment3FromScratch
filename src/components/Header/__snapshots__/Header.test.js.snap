// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render a header 1`] = `
<div
  className="header"
>
  <div
    className="first-row"
  >
    <div
      className="header-line"
    />
    <h1>
      BS
    </h1>
    <div
      className="header-line"
    />
  </div>
  <h2>
    The Book Shelf
  </h2>
</div>
`;
